chat/base.py

```diff
@@ -287,6 +287,14 @@ def prompt_config(checkpoint_dir: Path, tokenizer: Tokenizer) -> Tuple[str, Tupl
         )
         stop_tokens = ([tokenizer.eos_id],)
         return system_prompt, stop_tokens
+    
+    if re.search("NousResearch", checkpoint_name):
+        system_prompt = (
+            "### Instruction:\n{prompt}\n"
```

Comment:
Doesn't the the recommended prompt format leave a newline in between?

```suggestion
            "### Instruction:\n{prompt}\n\n"
```